pkg/lib/cockpit-components-listing.jsx

```diff
@@ -253,21 +255,30 @@ export class ListingRow extends React.Component {
                 );
             }
 
+            let simpleBody, heading;
```

Comment:
Will `let simpleBody` set `simpleBody` to null/undefined? (I am asking for my own information :) )

Reply:
Should be `undefined` yup